cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+  "baseUrl": "http://localhost:8080",
+  "video": false,
+  "viewportWidth": 1000,
```

Comment:
Those are quite strange viewport sizes. Can you use some standard resolutions?

Reply:
1920x1080 would be the best IMO.

Reply:
done. I'm using the Macbook 15 resolution (`1440x900`)

Reply:
Done.